src/modules/mavlink/mavlink_receiver.cpp

```diff
@@ -963,7 +963,7 @@ MavlinkReceiver::handle_message_vision_position_estimate(mavlink_message_t *msg)
 	// Use the component ID to identify the vision sensor
 	vision_position.id = msg->compid;
 
-	vision_position.timestamp_boot = hrt_absolute_time(); // Monotonic time
+	vision_position.timestamp_boot = sync_stamp(pos.usec);
```

Comment:
Essentially we want to keep the synced 'real' time in timestamp_computer and the time when the message was received in timestamp_boot. 

This is for measuring latency or sync quality in case we need it.